src/app/dataset-view/additional-components/flows-component/components/flows-table/flows-table.component.ts

```diff
@@ -41,26 +56,39 @@ export class FlowsTableComponent implements OnInit, OnChanges {
     @Input() public nodes: FlowSummaryDataFragment[];
     @Input() public filterByStatus: MaybeNull<FlowStatus>;
     @Input() public filterByInitiator: FilterByInitiatorEnum;
-    @Input() public searchByAccountName: string;
+    @Input() public searchByAccount: MaybeNull<Account>;
+    @Input() public searchByDataset: MaybeNull<Dataset>;
     @Input() public fetchStep: MaybeUndefined<FetchStep>;
     @Input() public transformData: TransformDescriptionTableData;
     @Input() public datasetBasics: DatasetBasicsFragment;
+    @Input() public accountView: boolean = false;
+    @Input() public accountName?: string;
+    @Input() public accountDatasets?: Dataset[];
```

Comment:
Design how you pass information about datasets universally - regardless of whether we are in dataset or account mode. The simple case for dataset page may use the same port, but contain just 1 dataset, i.e.

Reply:
You should have a universal port object that can:
 - give you `DatasetBasicsFragment` for given dataset id
 - same for `transformData` and `fetchStep`

Reply:
Created universal port object